m4/ax_compiler_version.m4

```diff
@@ -1,492 +0,0 @@
-# ===========================================================================
-#    http://www.gnu.org/software/autoconf-archive/ax_compiler_version.html
-# ===========================================================================
-#
-# SYNOPSIS
-#
-#   AX_COMPILER_VERSION
-#
-# DESCRIPTION
-#
-#   This macro retrieves the compiler version and returns it in the cache
-#   variable $ax_cv_c_compiler_version for C and $ax_cv_cxx_compiler_version
-#   for C++.
-#
-#   Version is returned as epoch:major.minor.patchversion
-#
-#   Epoch is used in order to have an increasing version number in case of
-#   marketing change.
-#
-#   Epoch use: * borland compiler use chronologically 0turboc for turboc
-#   era,
-#
-#     1borlanc BORLANC++ before 5, 2cppbuilder for cppbuilder era,
-#     3borlancpp for return of BORLANC++ (after version 5.5),
-#     4cppbuilder for cppbuilder with year version,
-#     and 5xe for XE era.
-#
-#   An empty string is returned otherwise.
-#
-# LICENSE
-#
-#   Copyright (c) 2014 Bastien ROUCARIES <roucaries.bastien+autoconf@gmail.com>
-#
-#   Copying and distribution of this file, with or without modification, are
-#   permitted in any medium without royalty provided the copyright notice
-#   and this notice are preserved. This file is offered as-is, without any
-#   warranty.
```

Comment:
This is even more relaxed license compared to BSD. No need to remove it.

Reply:
@lumag yes,  agree about that file.